libmamba/src/core/query.cpp

```diff
@@ -671,11 +787,25 @@ namespace mamba
 
     std::ostream& query_result::pretty(std::ostream& out) const
     {
-        if (!m_pkg_id_list.empty())
+        if (m_pkg_id_list.empty())
         {
+            out << "No entries matching \"" << m_query << "\" found" << std::endl;
+        }
+        else
+        {
+            std::map<std::string, std::vector<PackageInfo>> packages;
```

Comment:
Same thing, consider `undordered_map`.

Reply:
again, we need to maintain the order here, hence the `map`.